src/hangar/backends/selection.py

```diff
@@ -22,45 +22,47 @@
 convention, the first digit of the code can be used to identify the storage
 medium:
 
-    * Lowercase ascii letters & digits [0, 1, 2, 3, 4] -> reserved for
+   *  Lowercase ``ascii_letters`` & digits ``[0, 1, 2, 3, 4]`` -> reserved for
       backends handling data on the local disk.
-    * Uppercase ascii letters & digits [5, 6, 7, 8, 9] -> reserved for
+
+   *  Uppercase ``ascii_letters`` & digits ``[5, 6, 7, 8, 9]`` -> reserved for
       backends referring to data residing on a remote server.
 
 This is not a hard and fast rule though, and can be changed in the future if the
 need arises.
 
 Process & Guarantees
----------------------
+--------------------
 
 In order to maintain backwards compatibility across versions of Hangar into the
 future the following ruleset is specified and MUST BE HONORED:
 
-    * When a new backend is proposed, the contributor(s) provide the class with a
-      canonical name (HDF5_00, TILEDB_01, etc) for developer consumption in the
-      backend. The review team will provide an available two-digit code which all
-      records corresponding to that backend must identify themselves with.
+*  When a new backend is proposed, the contributor(s) provide the class with a
+   canonical name (``HDF5_00``, ``TILEDB_01``, etc) for developer consumption in
```

Comment:
To make it lighter in the future (when we'll need to squeeze out performance), we could impose the human-readable code to be fixed-width, like 9 chars `HDF5___00` and `TILEDB_01`, or 7 chars `HDF5_00` and `TLDB_01`.